catalog/transformer/catalog-transformer-xml/src/test/java/ddf/catalog/transform/xml/TestXmlValidationEventHandler.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transform.xml;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import ddf.catalog.transformer.xml.XmlValidationEventHandler;
+import javax.xml.bind.ValidationEvent;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestXmlValidationEventHandler {
```

Comment:
Just realized that this class breaks our unit test naming convention. Might be worth fixing for master only since we just went through and fixed all of them.

Reply:
The funny thing about this is IntelliJ put Test at the end, but on the release branch they all had it at the beginning so I matched that and thought I had it correct 🤦‍♂️